usaspending_api/spending/v2/filters/fy_filter.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+
+from usaspending_api.common.exceptions import InvalidParameterException
+
+
+def fy_filter(fy, now):
+
+    # Define fiscal reporting quarters
+    q1_start = datetime.strptime('10-01', '%m-%d').strftime('%m-%d')
+    q1_end = datetime.strptime('12-31', '%m-%d').strftime('%m-%d')
+    q2_start = datetime.strptime('01-01', '%m-%d').strftime('%m-%d')
+    q2_end = datetime.strptime('03-31', '%m-%d').strftime('%m-%d')
+    q3_start = datetime.strptime('04-01', '%m-%d').strftime('%m-%d')
+    q3_end = datetime.strptime('06-30', '%m-%d').strftime('%m-%d')
+    q4_end = datetime.strptime('09-30', '%m-%d').strftime('%m-%d')
+
+    # Validate fiscal year and return fiscal_quarter and fiscal_date fro results
```

Comment:
This may be automated to convert to a fiscal quarter. What is the intended use of "fiscal_date"?

Reply:
The results output asks for it.
{
    "total": 5027345283.72,
    "end_date": "2017-06-30",
    "results": [
        {

Reply:
According to the gist